lib/modules/commentDepth.js

```diff
@@ -39,9 +44,14 @@ module.options = {
 		}, {
 			name: 'commentDepth',
 			type: 'text',
+			value: '4',
+		}, {
+			name: 'minimumComments',
+			type: 'text',
+			value: '50',
```

Comment:
In a larger module I'd say no because it might be easy to miss, but this module is so small that it's not a big deal